MVend/GwizaDash
# ft(): add authentication

### What does this PR do?

- [x] Add authentication
- [x] Refactor code to add authentication
- [x] Add logout functionality

### How to test this?

- Check out this branch `ft-authentication`
- Do `yarn` to install packages
- Then do `yarn start`

### Expected behavior

- You should land on the login page
- After successfully logging in you should be redirected to the dashboard 
- If logged out you can go back without logging in, it redirects back to the login page
